src/utils/auth/isEmailValid.ts

```diff
@@ -0,0 +1,5 @@
+export const isEmailValid = (email: string): boolean => {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
Zod's `email: z.string().email()` probably should be enough.

Reply:
Potentially an outdated code.

Reply:
removed